packages/cssnano/src/__tests__/postcss-minify-selectors.js

```diff
@@ -79,7 +79,7 @@ test(
   'should handle deep combinators',
   processCss(
     'body /deep/ .theme-element{color:#00f}',
-    'body /deep/ .theme-element{color:#00f}'
+    'body/deep/.theme-element{color:#00f}'
```

Comment:
I think this is correct because the parser now recognizes [named combinators](https://github.com/postcss/postcss-selector-parser/blob/master/CHANGELOG.md#support-for-named-combinators) so is able to properly remove spaces around them.